commands/prioritize.py

```diff
@@ -128,7 +130,9 @@ def __init__(
         Raises:
         ConnectionError: If no available port is found.
         """
-        assert len(embeddings) > 0 and len(embeddings[0]) > 0
+        assert (
```

Comment:
can you add a reference of the method used for the prioritization in the docstring?

Reply:
?